CCFatigue_modules/3_CLD/Harris/cld_harris.py

```diff
@@ -25,12 +26,110 @@
 # staticvalue.txt => constants
 UCS = 27.1
 UTS = 27.7
-CRITICAL_STRESS_RATIO = 0.1
+
+BOUNDS_MARGIN = 0.99
 
 # Cycles for the isolines (the lines of the CLD)
 CYCLES_COUNT = [10**x for x in range(3, 10)]  # = 1e3, 1e4, ..., 1e9
 
 
+def calculate_stress_amplitude(stress_ratio: float, max_stress: float) -> float:
+    """
+    Calculate stress_amplitude (sigma_a) for given stress ratio (R) and max stress
+    Src: https://github.com/EPFL-ENAC/CCFatiguePlatform/blob/develop/CCFatigue_modules/3_CLD/Harris/CLD-Harris.for#L68
+    - lines 68-74
+    Inputs:
+    - stress_ratio (R)
+    - max_stress (sigma_max)
+    Output:
+    - stess amplitude sigma_a
+    """
+    if abs(stress_ratio) > 1:
+        stress_amplitude = (1 - (1 / stress_ratio)) * max_stress / 2
+    else:
+        stress_amplitude = (1 - stress_ratio) * max_stress / 2
+    return stress_amplitude
+
+
+def calculate_stress_mean(stress_ratio: float, max_stress: float) -> float:
+    """
+    Calculate mean stress (sigma_mean) for given stress ratio (R) and max stress
+    Src: https://github.com/EPFL-ENAC/CCFatiguePlatform/blob/develop/CCFatigue_modules/3_CLD/Harris/CLD-Harris.for#L68
+    - lines 68-74
+    Inputs:
+    - stress_ratio (R)
+    - max_stress (sigma_max)
+    - uts
+    - ucs
+    Output:
+    - mean stress sigma_mean
+    """
+    if abs(stress_ratio) > 1:
+        sigma_mean = (1 - (1 / stress_ratio)) * max_stress / 2
+    else:
+        sigma_mean = (1 + stress_ratio) * max_stress / 2
+
+    return sigma_mean
+
+
+def bounds_stress_mean(sigma_mean, uts: float, ucs: float, margin: float) -> float:

```

Comment:
I don't understand why this is not done within the function `calculate_stress_mean`

Reply:
calculate_stress_mean is applied in both module 3 and 5 but bounds_stress_mean is applied only in module 3.
I need to create a harris_utils.py for common resources to both modules instead of importing module 3 in module 5, it will be more clear.